CONTRIBUTING.md

```diff
@@ -190,6 +192,8 @@ feature. In general, the team prioritizes code reviews over other work.
 * Merged code is automatically deployed to the [staging
   environment](https://michigan-benefits-staging.herokuapp.com/) after a green
 build on Circle CI.
+* Once code completing a Tracker story is deployed to staging, mark the story as 'Delivered'. This 
```

Comment:
this is true for features, but not for chores. Not sure about bugs. Should we explain the difference?